apps/examples/testcase/ta_tc/wifi_manager/utc/Kconfig

```diff
@@ -0,0 +1,71 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC
+	bool "Wi-Fi Manager UTC TestCase Example"
+	select WIFI_MANAGER
+	default n
+	---help---
+		Enable the Wi-Fi Manager TestCase Example
+
+if EXAMPLES_TESTCASE_WIFI_MANAGER_UTC
+comment "Specify your AP which you want to connect to"
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_SSID
+    string "SSID of AP"
```

Comment:
Please use a same indentation. In others, tab is used.

Reply:
I revise this.